README.md

```diff
@@ -2,19 +2,134 @@
 
 ## Bring 'Code as Data' to market with enterprises
 
+### Enable engine integration with 3rd parties
+
+- [ ] gitbase-spark-connector SQL 3rd party interop (Spark Thrift Server) [P0][DP]
+
+### Increase number of algorithms we can run
+
+- [ ] Support for diff tree [P0][DP]
```

Comment:
We should specify that "diff tree" here is the difference between file trees in Git, not source code trees.